changelogs/unreleased/6608-shawn-hurley

```diff
@@ -0,0 +1 @@
+Design update to handle block devices during the kopia data mover
```

Comment:
Update the change log to something like "Add API support for volMode block" or something similar" or "block mode mover impl part:1 "

Reply:
updated